frontend/src/metabase/css/core/fonts.css

```diff
@@ -477,3 +477,20 @@
   font-style: normal;
   font-display: swap;
 }
+
+@font-face {
+  font-family: "Fira Code";
```

Comment:
❤️

Reply:
I'm sorry, but it won't make it in 😅

Reply:
![giphy](https://user-images.githubusercontent.com/30528226/212742451-1ed60820-e893-45c8-a132-fac56312682a.gif)